provisioner/components/k8s/validate_playbook/cluster_validate_playbook.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: master-node
+  become: true
+  roles:
+    - role: validate_cluster
```

Comment:
are some files showing the red sign as no newline at the end?
so please add a new line at the end wherever such line can be seen.

Reply:
Fixed it